packages/devtools_app/lib/src/debugger/debugger_controller.dart

```diff
@@ -200,6 +200,7 @@ class DebuggerController extends DisposableController
 
   /// Jump to the given ScriptRef and optional SourcePosition.
   void showScriptLocation(ScriptLocation scriptLocation) {
+    toggleFileOpenerVisibility(false);
```

Comment:
I'm surprised the visibility toggles are needed with `onFocusLost: _onClose,` below

Reply:
Agreed -  I added this so that selecting a file in the program explorer will also close the file opener. Currently the program explorer never gets focus. I tried adding focusNodes to it to automatically resolve this, but I wasn't able to get it working. I've added a TODO with details.